e2e/playwright/info-drawer/src/tests/comments.spec.ts

```diff
@@ -0,0 +1,153 @@
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import { ApiClientFactory, Utils, test, TrashcanApi, FavoritesPageApi, NodesApi, FileActionsApi } from '@alfresco/playwright-shared';
+
+test.describe('Info Drawer - Comments', () => {
+  const apiClientFactory = new ApiClientFactory();
+  let nodesApi: NodesApi;
+  let trashcanApi: TrashcanApi;
+  let fileActionsApi: FileActionsApi;
+  let favoritesActions: FavoritesPageApi;
+  const username = `user-e2e-${Utils.random()}`;
+  const commentText = `comment-e2e-${Utils.random()}`;
+  const persFolderName = `persFolder-e2e-${Utils.random()}`;
+  const favFolderName = `favFolder-e2e-${Utils.random()}`;
+  const sharedFileName = `sharedFile-e2e-${Utils.random()}`;
+
+  test.beforeAll(async () => {
+    try {
+      await apiClientFactory.setUpAcaBackend('admin');
+      await apiClientFactory.createUser({ username });
+      nodesApi = await NodesApi.initialize(username, username);
+      trashcanApi = await TrashcanApi.initialize(username, username);
+      favoritesActions = await FavoritesPageApi.initialize(username, username);
+      fileActionsApi = await FileActionsApi.initialize(username, username);
+    } catch (error) {
+      console.error(`beforeAll failed : ${error}`);
+    }
+  });
+
+  test.beforeEach(async ({ loginPage }) => {
+    try {
+      await loginPage.loginUser({ username: username, password: username }, { withNavigation: true, waitForLoading: true });
+    } catch (error) {
+      console.error(`Main beforeEach failed: ${error}`);
+    }
+  });
+
+  test.afterAll(async () => {
+    try {
+      await nodesApi.deleteCurrentUserNodes();
+      await trashcanApi.emptyTrashcan();
+    } catch {}
+  });
+
+  test('[C299173] from Personal Files - Comments tab default fields', async ({ personalFiles }) => {
+    await nodesApi.createFolder(persFolderName);
+    await fileActionsApi.waitForNodes(persFolderName, { expect: 1 });
+    await personalFiles.navigate();
+    await expect(personalFiles.dataTable.getRowByName(persFolderName)).toBeVisible();
+    await personalFiles.dataTable.getRowByName(persFolderName).click();
+    await personalFiles.acaHeader.viewDetails.click();
+    await personalFiles.infoDrawer.commentsTab.click();
+    await expect(personalFiles.infoDrawer.commentInputField).toBeVisible();
+    expect(await personalFiles.infoDrawer.checkCommentsHeaderCount()).toEqual(0);
+    expect(await personalFiles.infoDrawer.addCommentButton).toBeDisabled();
```

Comment:
Are you sure you put 'await' in correct place? Looks like toBeDisabled function returns Promise so await should be put before expect

Reply:
You are right, I should place it before expect here. Will be fixed in the next push.